client/a8c-for-agencies/sections/purchases/payment-methods/payment-method-add/payment-method-form/index.tsx

```diff
@@ -192,6 +192,11 @@ function PaymentMethodForm() {
 						returnQueryArg
 					)
 				);
+			} else if ( returnQueryArg?.includes( 'new_dev_site' ) ) {
```

Comment:
In the code above we are setting the parameter to `add_new_dev_site=true`, but here we are checking for `new_dev_site` and then setting the parameter to `add_new_dev_site: true` again.

We could consider using a single parameter everywhere, e.g. `add_new_dev_site`.

```suggestion
			} else if ( returnQueryArg?.includes( 'add_new_dev_site' ) ) {
```

Reply:
You are 100% right, lets move it all to `add_new_dev_site`